desktop/apps/galleries_institutions/components/primary_carousel/view.coffee

```diff
@@ -22,7 +22,11 @@ module.exports = class PrimaryCarousel extends Backbone.View
   fetch: (params) ->
     promise = fetchProfiles(params.toJSON())
     promise.then (profiles) =>
-      @profiles.reset profiles.models
+      if profiles?.models?.length
+        @$el.fadeIn 500
+        @profiles.reset profiles.models
+      else
+        @$el.hide()
```

Comment:
An alternative may be to do a `length` check in the [template](https://github.com/artsy/force/blob/aac0b547c620336e42a211c80a53d35b052a862b/desktop/apps/galleries_institutions/components/primary_carousel/template.jade#L8), so we don't need to show/hide additionally. But I could be wrong here.